d2l-button-subtle.html

```diff
@@ -97,28 +97,16 @@
 				padding-left: 0;
 				padding-right: 1.2rem;
 			}
-			:host-context([dir="rtl"])[icon] .d2l-button-subtle-content {
-				@apply --content-icon-rtl;
-			}
-			:host-context([dir="rtl"]):host([icon]) .d2l-button-subtle-content {
-				@apply --content-icon-rtl;
-			}
 			:host(:dir(rtl))[icon] .d2l-button-subtle-content {
 				@apply --content-icon-rtl;
 			}
-			:host(:host-context([dir="rtl"])[icon]) .d2l-button-subtle-content {
+			:host([dir="rtl"][icon]) .d2l-button-subtle-content {
```

Comment:
This should be `:host([dir="rtl"])[icon]` or `:host([dir="rtl"]):host([icon])`

Reply:
`:host([dir="rtl"])` will only work if the `dir` attribute is set on the component itself

Reply:
`:host(:dir(rtl))` or `:host-context([dir="rtl"])` (outside of the `:host` selector)